frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-items.tsx

```diff
@@ -522,6 +523,7 @@ export const OperatorHubTileView: React.FC<OperatorHubTileViewProps> = (props) =
                       'pf-c-button',
                       { 'pf-m-secondary': remoteWorkflowUrl },
                       { 'pf-m-primary': !remoteWorkflowUrl },
+                      { 'pf-m-disabled': detailsItem.isInstalling },
```

Comment:
I think this is in the wrong place. If the operator is not "installed", then it can't be "installing" either.

We probably need the disabled state to be on the "Uninstall" action below:
```
<Button
  // ...
  isDisabled={detailsItem.isInstalling}
  // ...
/>
```

Reply:
Actually, I don't think the uninstall action should be disabled on second thought. But currently, there is no way for the `Install` button to be showing in a disabled state because the 'isInstalling' state depends (or should) on the existence of the Subscription. An operator cannot be pending install if no subscription has been created yet.